src/pages/using-spark/components/button.mdx

```diff
@@ -94,18 +82,40 @@ Represents the main call to action on the page. Use no more than two Primary But
   titleAttr="Primary Button Example"
 />
 
-### Secondary Button
+#### Usage
+
+- Use to point out the main call to action on an interface.
+- Use one Primary (Filled) button per screen to help users understand the action hierarchy of the page and to avoid any confusion.
+- In a process flow, use to show the main action to help users navigate to the next step.
 
-Represents the second-highest priority action, following the Primary Button. There can be more than one on a page.
+**Example:**
 
-#### Examples
-- Preview
-- Save For Later
-- Back
-- In a multi-step form, a Secondary Button works with Primary and Tertiary Button as a set of related actions.
-  - Primary Button is "Submit"
-  - **Secondary Button is "Back"**
-  - Tertiary Button is "Cancel"
+<img
+  srcSet={buttonModalSet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbm sprk-u-Width-100"
+  src={buttonModal}
+  alt="A primary button is on the right and represents the main action"
```

Comment:
This alt text needs a period at the end.

Reply:
Added and also capitalized "Primary Button"